### DISCUSSION TITLE
Types of Careers: Government Jobs

### DISCUSSION THREAD

**Discussion by Justin-Bento** [TheOdinProject/curriculum#24209] [Q&A]
Hey Odin Project Team,

I have a question regarding the types of carrers, for the introduction of web development.

Why didn't you mention applying for government jobs, while I am aware each government has their own requirements. 

Shouldn't it at least be an option, as not everyone would want to work in the other envirornments due to lack of support.

Would love to hear your throughts.

**Comment by ChargrilledChook** [comment]
Hi @Justin-Bento, can you point me towards the lesson you're referring to? I think broadly, being a developer is very similar work if it's working for a startup, an enterprise or the government but hard to say without knowing a bit more context

**Reply by Justin-Bento** [reply]
Sure thing, the lesson is [Introduction to Web Development](https://www.theodinproject.com/lessons/foundations-introduction-to-web-development) and the main headline is Types Of Carrers futher down the page.

**Reply by ChargrilledChook** [reply]
Thanks for getting back to me. I think government jobs are covered by the last section (larger, older companies) but maybe a word or two could be added in there to make that more obvious. What do you think? Would you like to make that change?

**Reply by Justin-Bento** [reply]
Sorry for the late response, took a break over the summer but I was going to point to here: 

> Finally, large, older companies still need web developers. These companies offer a good work/life balance, pay, and benefits but often move slower than a company that is highly focused on tech.

How would the software development lifecycle be different from a fast paced environment? As a startup normally has less restrictions compared to a slower company and they and will make you wear many hats. Do government Software Development jobs make you organize, testing, and document before actually writing the code. As I'm curious as to what a government would want from a web developer compared to a normal company.
